.github/workflows/tests.yml

```diff
@@ -67,7 +67,7 @@ jobs:
       - name: Setup node with self-hosted registry
         uses: actions/setup-node@v3
         with:
-          node-version: "18"
+          node-version: "16"
```

Comment:
why? i thought flox is moving to 18 node

could you pleass put up a comment, motivating the version?

Reply:
cli wasn't released for the node 18 and I broke everything